storages/sled-storage/tests/sled_transaction.rs

```diff
@@ -30,16 +30,25 @@ macro_rules! exec {
 
 macro_rules! test {
     ($glue: ident $sql: literal, $result: expr) => {
-        assert_eq!($glue.execute($sql), $result);
+        let res = $glue.execute($sql);
+        let expeced_result: Result<Payload> = $result;
+        if let Ok(r) = res {
+            assert_eq!(expeced_result.is_ok(), true);
+            assert_eq!(r.len(), 1);
+            assert_eq!(r[0], expeced_result.unwrap());
+        } else {
+            assert_eq!(res.err(), expeced_result.err());
+        }
```

Comment:
```suggestion
        let actual = $glue.execute($sql);
        assert_eq!(actual, $expected.map(|payload| vec![payload]));
```

Let's change it simply!

Reply:
done